engine/gamesys/src/gamesys/components/comp_sound.h

```diff
@@ -19,25 +19,7 @@
 
 namespace dmGameSystem
 {
-    dmGameObject::CreateResult CompSoundNewWorld(const dmGameObject::ComponentNewWorldParams& params);
```

Comment:
The script system doesn't interact directly with the component type, and these lifecycle can all be private.